draft-ietf-core-cf-reg-update.md

```diff
@@ -196,16 +196,18 @@ A new column with the title "Notes" has been added to the CoAP Content-Formats R
 
 ## New Section 12.3.1 "Temporary Content-Format Registrations"
 
-This section clarifies that the "CoAP Content-Formats" registry allows temporary registrations within the 0-255 and 256-9999 ranges.
-The range 10000-64999 does not allow temporary registrations.
+This section clarifies that the "CoAP Content-Formats" registry allows temporary registrations within the 0-255, 256-9999 and 10000-64999 ranges.
 
 A temporary registration may be created for example by an IANA early allocation action, as requested by the authors of an Internet-Draft in the IETF stream.
-Alternatively, it may be created because the referenced media type is still provisional (that is, included in the IANA "Provisional Standard Media Type" registry {{IANA.provisional-standard-media-types}}).
+If the referenced media type is provisional (that is, included in the IANA "Provisional Standard Media Type" registry {{IANA.provisional-standard-media-types}}) then a temporary registration is always created.
 
-A temporary registration is marked by IANA with the label "TEMPORARY" in the corresponding registry entry.
-Once the required review procedure for the temporary ID has successfully completed, and the referenced media type is included in the IANA Media Types registry {{IANA.media-types}}, IANA must remove the "TEMPORARY" label so that the entry becomes permanent.
-If the requested temporary entry does not successfully pass its required review procedure, IANA must remove the entry again and set the Content-Format ID value back to "Unassigned".
-This may happen for example when an Internet-Draft requesting a Content-Format ID is abandoned, or when the referenced provisional media type is abandoned.
+A temporary registration is marked as such by IANA in the corresponding registry entry.
+Once the required review procedure for the temporary ID has successfully completed, and the referenced media type is included in the IANA Media Types registry {{IANA.media-types}}, IANA must remove any indication about the temporary nature of the registration so that the entry becomes permanent.
+
+If a temporary registration in the range 0-9999 does not successfully pass its required review procedure, IANA must remove the entry again and set the Content-Format ID value back to "Unassigned".
```

Comment:
OLD
> If a temporary registration in the range 0-9999 does not

NEW
> If a temporary registration (in any range) does not

The changes above are admitting temporary registrations also for identifiers in the range 10000-64999, i.e., in all ranges, right?

Reply:
It's similar to a registration via email for example: it gets added, and maybe no-one will ever use it anymore, but still the ID remains forever as long as the media type exists.

Reply:
I see, thanks! Then two things:

* "in the range 0-9999 does not" can instead be "in the 0-255 and 256-9999 ranges", since the registry shows those two as independent "ranges".

* Maybe it's better to include this explanation in a follow-up sentence, as a note that entries in the 10000-64999 range are going to stay (unless, as explained right below, they correspond to a provisional media type and this gets abandoned, at which point they are also removed).

Reply:
Ok I can try an update for this!